src/EndpointV2/EndpointV2Middleware.php

```diff
@@ -303,4 +303,11 @@ private function normalizeAuthScheme(array $authScheme) : array
 
         return $normalizedAuthScheme;
     }
+
+    private function appendAccountIdParameter(&$commandArgs)
```

Comment:
I think we need to have the account id mode available in the args.  Identity might be resolved in other places and not always because of account id retrieval, so I think this should be done conditionally.